### PR TITLE
FB-2 Refactor FizzBuzzApplication

### PR DESCRIPTION
**Pull request by Nika-Avalishvili**
I Refactored FizzBuzzApplication

### CODE REVIEW

File: src/main/java/com/example/fizzbuzz/controller/FizzBuzzDBController.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+        Connection connection = null;
+        Statement statement = null;
+
+        ConnectFizzBuzzDB objConnectFizzBuzzDB= new ConnectFizzBuzzDB();
+        connection = objConnectFizzBuzzDB.get_connection();
+
+
+        String result = "";
+        result = setResult(myNumber, result);
```
Comment (yuriy-osychenko) [inline src/main/java/com/example/fizzbuzz/controller/FizzBuzzDBController.java]: This code seems to be repeated in each method, can we extract it to new method and reuse it everywhere?

Comment (Nika-Avalishvili) [reply src/main/java/com/example/fizzbuzz/controller/FizzBuzzDBController.java]: I committed new version of code on this branch

File: src/main/java/com/example/fizzbuzz/controller/FizzBuzzDBController.java [hunk 2]
```diff
@@ -22,90 +22,46 @@ public static String setResult(Integer myNumber, String result){
     }
 
 
-    @GetMapping("/insertValue")
-    public static void addValues(@RequestParam(value = "id") Long id, @RequestParam (value = "myNumber") Integer myNumber, @RequestParam (value = "name") String name) {
-
+    public static void connectionAndExecute(String query, Integer myNumber){
```
Comment (yuriy-osychenko) [inline src/main/java/com/example/fizzbuzz/controller/FizzBuzzDBController.java]: Do we need `myNumber` in this method?
It seems like it is not used and we can just remove it and also we won't need to execute this method with 0s where myNumber is not needed

Comment (Nika-Avalishvili) [reply src/main/java/com/example/fizzbuzz/controller/FizzBuzzDBController.java]: You are right, I missed that it was not even used anywhere

File: src/main/java/com/example/fizzbuzz/controller/FizzBuzzDBController.java [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        ConnectFizzBuzzDB objConnectFizzBuzzDB= new ConnectFizzBuzzDB();
+        connection = objConnectFizzBuzzDB.get_connection();
+
+        try {
+            statement = connection.createStatement();
+            statement.executeUpdate(query);
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
```
Comment (yuriy-osychenko) [inline src/main/java/com/example/fizzbuzz/controller/FizzBuzzDBController.java]: I think these 2 methods contain logic which is not related to mapping itself, what do you think about moving these 2 methods to some new services?
For example `FizzBuzzResultService` with setResult method and `FizzBuzzDBService` with connectionAndExecute method

**Review by yuriy-osychenko — APPROVED** [review]

**Review by yuriy-osychenko — APPROVED** [review]

**Review by yuriy-osychenko — APPROVED** [review]

**Pull request merged**